nav2_bringup/urdf/gz_turtlebot3_waffle.urdf

```diff
@@ -0,0 +1,423 @@
+<?xml version="1.0" ?>
+<robot name="turtlebot3_waffle" xmlns:xacro="http://ros.org/wiki/xacro">
+  <!-- <xacro:include filename="$(find turtlebot3_description)/urdf/common_properties.urdf"/>
+
+  <xacro:property name="r200_cam_rgb_px" value="0.005"/>
```

Comment:
Is this not available in a debian package we can pull in instead? Our TB3 model for Gazebo is historical due to the initial TB3 models being really not great in a series of ways, but I'm not sure if the new ones are finally usable - I suspect they finally fixed the majority of their issues. So I'd prefer not to add more code here if we can avoid it

Reply:
These lines are in the `turtlebot3_waffle.urdf` too. I just copy-paste the file and adapted to the new gazebo. Removed

Reply:
I don't mean those lines, I mean this entire file :smile: I can't commend on a file yet in GitHub's UX so I just picked my current spot scrolling, sorry hah

Reply:
If we can use the URDF from the TB3 binaries, that would be optimal to not ingest more source files here. We originally had this here because the sensor mounting locations were bust on the TB3 URDF that we had to manually change + the meshes were insanely overly detailed so we replaced with basic shapes. I believe (?) that is fixed now so its worth seeing if we can avoid that hack moving forward into the new simulator.

Anyway, we'll be replacing the robot with another robot soon enough so I suppose I don't care all that greatly. Just a suggestion if it would make our lives easier